acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
I'm not sure about these changes to the gossip peerings test. I think if a test is expecting node events like this it should handle them like this instead of trying to fit within the `AssertAndStop` interface.